report/chapters/introduction.typ

```diff
@@ -8,3 +8,6 @@ The process of finding a collaborative company began by searching through the co
 The aim of this project is to develop a functional application that prioritizes user experience and meets the client’s needs. The primary requirement for this system is to develop a holiday planner to track non-working days for the company’s employees. KMD already has a holiday planner, but since it’s outdated, they want to replace it with an imporved version. 
 
 == Current System at KMD
+Aside from using Microsoft Outlook Calendar and the calendar integrated within their planning boards, KMD currently have an outdated vacation planning solution called KMDFerieplan. This system, developed approximately 15 years ago, operates on a Microsoft backend and is integrated with SAP SuccessFactors for managing vacation codes. Previously, it could automatically synchronize vacation requests with the calendar, but this functionality is no longer available following KMD's migration to SAP S/4HANA, as the system is incompatible with the updated enterprise resource planning (ERP) platform.
```

Comment:
```suggestion
Aside from using Microsoft Outlook Calendar and the calendar integrated within their planning boards, KMD currently have an outdated vacation planning solution called KMDFerieplan. This system, developed approximately 10 years ago, operates on a Microsoft backend and is integrated with SAP SuccessFactors for managing vacation codes. Previously, it could automatically synchronize vacation requests with the calendar, but this functionality is no longer available following KMD's migration to SAP S/4HANA, as the system is incompatible with the updated enterprise resource planning (ERP) platform.
```

Reply:
Mats said it was developed in 2014